src/Standards/Generic/Sniffs/PHP/SyntaxSniff.php

```diff
@@ -61,14 +61,9 @@ public function process(File $phpcsFile, $stackPtr)
         }
 
         $fileName = escapeshellarg($phpcsFile->getFilename());
-        if (defined('HHVM_VERSION') === false) {
-            $cmd = escapeshellcmd($this->phpPath)." -l -d display_errors=1 -d error_prepend_string='' $fileName 2>&1";
-        } else {
-            $cmd = escapeshellcmd($this->phpPath)." -l $fileName 2>&1";
-        }
-
-        $output  = shell_exec($cmd);
-        $matches = [];
+        $cmd      = escapeshellcmd($this->phpPath)." -l $fileName 2>&1";
```

Comment:
AFAICS, this should be `$cmd = escapeshellcmd($this->phpPath)." -l -d display_errors=1 -d error_prepend_string='' $fileName 2>&1";`

Reply:
I figured as much ;-)